components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -51,6 +54,26 @@
 static NSString *const kAccessibilityIdentifierKeyPath = @"accessibilityIdentifier";
 static NSString *const kAccessibilityTraitsKeyPath = @"accessibilityTraits";
 
+/** The internal layout styles of the tab bar. */
+typedef NS_ENUM(NSUInteger, MDCTabBarViewInternalLayoutStyle) {
+
+  /** Items are laid-out from leading to trailing and sized to their intrinsic content size. */
+  MDCTabBarViewInternalLayoutStyleScrollable = 0,
+  /**
```

Comment:
Nit: New line here or remove the extra line in the other documenation for the enum types.

Reply:
Done